.github/workflows/source_azuredevops.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         run: make gen
       - name: Fail if generation updated files
         if: github.event_name == 'pull_request'
-        run: test "$(git status -s | wc -l)" -eq 0
+        run: test "$(git status -s | wc -l)" -eq 0 || (git status -s; exit 1)
```

Comment:
what does the new addition here do? 🤔

Reply:
It prints the list of modified files (we have it in other plugins too)

Reply:
The purpose is to see which files changed instead of only the error:
![image](https://github.com/cloudquery/cloudquery/assets/26760571/20369292-1a8b-4c77-9921-2dd73362d459)